karapace/coordinator/schema_coordinator.py

```diff
@@ -227,6 +231,9 @@ def check_errors(self) -> None:
         if self._pending_exception is not None:
             exc = self._pending_exception
             self._pending_exception = None
+            if (exc in self.SCHEMA_COORDINATOR_RETRIEABLE_ERRORS) or (getattr(exc, "retriable", False)):
```

Comment:
Should we also use something like, [coordinator_dead()](https://github.com/Aiven-Open/karapace/blob/main/karapace/coordinator/schema_coordinator.py#L429) or [request_rejoin()](https://github.com/Aiven-Open/karapace/blob/main/karapace/coordinator/schema_coordinator.py#L447), ideally the coordinator will retry some errors.